### DISCUSSION TITLE
Do I need to call store.Commit after only calling store.Read and want to end the transaction?

### DISCUSSION THREAD

**Discussion by Zacama** [open-policy-agent/community#438] [OPA and Rego] [answered]
Hello, I am new to using OPA. I have a problem integrating OPA with Go.
Do I need to call store.Commit after only calling store.Read and want to end the transaction?

**Comment by charlieegan3** [comment, marked as answer]
Hey again, yes that's my understanding.

**Reply by Zacama** [reply]
https://pkg.go.dev/github.com/open-policy-agent/opa/storage#Store
Thank you very much. I think I understand it.
